### PR TITLE
Big Picture instructions

### PR DESCRIPTION
**Pull request by dbampalikis**
- Change the instructions to support for the Big Picture submission
- Added the Big Picture crypt4gh public key

Given that the instructions are duplicated in the website, should we merge this pull request or share this branch instead?

### CODE REVIEW

**Review by kostas-kou — COMMENTED** [review]
Looks fine to me

**Review by kostas-kou — APPROVED** [review]

**Review by norling — APPROVED** [review]

**Review by shreyasshivakumara — APPROVED** [review]
Looks good!

File: README.md [hunk 1]
```diff
@@ -73,26 +77,27 @@ Outside of Bianca, this can be limited to these two or three steps:
    `-s` argument.
 
    ```bash
-   ./crypt4gh encrypt -p crypt4gh_key.pub -f [my-file] [-s [my-key].sec.pem]
+   ./crypt4gh encrypt -p [crypt4gh-public-key.pub] -f [my-file] [-s [my-key].sec.pem]
    ```
+   where `crypt4gh-public-key.pub` the key crypt4gh public key you downloaded in the previous step.
```
Comment (aaperis) [inline README.md]:
```suggestion
   where `crypt4gh-public-key.pub` is the crypt4gh public key you downloaded in the previous step.
```

File: README.md [hunk 2]
```diff
@@ -47,17 +47,21 @@ Outside of Bianca, this can be limited to these two or three steps:
...
    For the SDA to be able to understand the encrypted files when they are
    uploaded, they need to be encrypted with the correct public key. This key can
-   be downloaded from this repository with this command:
+   be downloaded from this repository with this command <strong>if you are uploading to the SDA</strong>:
    ```bash
    wget https://raw.githubusercontent.com/NBISweden/EGA-SE-user-docs/main/crypt4gh_key.pub
    ```
+   or this command <strong>if you are uploading to Big Picture</strong>:
+   ```bash
+   wget https://raw.githubusercontent.com/NBISweden/EGA-SE-user-docs/main/crypt4gh_bp_key.pub
```
Comment (aaperis) [inline README.md:56]: download didn't work (while the one for SDA worked)

**Review by aaperis — CHANGES_REQUESTED** [review]
looks good but couldn't download bp key

File: README.md (hunk 2 above)
Comment (dbampalikis) [reply README.md:56]: I think that's because the PR is not merged, I will check it once that's done

**Review by aaperis — APPROVED** [review]

**Pull request merged**